plenigo_plugin/settings/SettingCurtainButtonCatCustom.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+  Copyright (C) 2014 plenigo
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ */
+
+namespace plenigo_plugin\settings;
+
+/**
+ * Setting class for curtain_cat_custom_title
```

Comment:
This means "Setting class for the 'curtain_cat_custom_title' option stored" so the option name and the class doesnt have to match